server.key

```diff
@@ -0,0 +1,15 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Committing a private key??? Looks like this was accidental; hope it's not much trouble for you to regenerate it. I'll delete it after merging, same with `server.crt`.

Was this to set up https, because the prod build on your desktop kept redirecting to https? `?allowinsecure=1` allows http. Even works in production: http://swarmsim.github.io/#/?allowinsecure=1

(https and http use different cookies/localstorages, so if one of the two doesn't redirect, players would panic because their save "disappeared".)